src/test/java/htsjdk/samtools/util/IOUtilUnitTest.java

```diff
@@ -100,7 +92,7 @@ public void testGetCanonicalPath() throws IOException {
         }
 
         File tmpDir = new File(tmpPath, userName);

```

Comment:
Since you are changing this code, can you verify that this is necessay? why we need to use the actual user name for such a test a supposed to random subdir name.